build-tools-internal/src/main/resources/templates/breaking-changes.asciidoc

```diff
@@ -7,7 +7,7 @@
 This section discusses the changes that you need to be aware of when migrating
 your application to {es} ${majorDotMinor}.
 
-See also <<release-highlights>> and <<es-release-notes>>.
+See also {ref-bare}/${majorDotMinor}/release-highlights.html[What's new in ${majorDotMinor}] and <<es-release-notes>>.
```

Comment:
> See also {ref-bare}/${majorDotMinor}/release-highlights.html[What's new in ${majorDotMinor}] and <<es-release-notes>>.

Another option to consider might be to add a "next-major" attribute to the docs repo too (e.g. like the prev-major-* attributes we already have in https://github.com/elastic/docs/blob/master/shared/versions/stack/8.6.asciidoc.  That way it'd be controlled uniformly across the library and this URL could just be  "{ref-bare}/{next-major}/..." or something along those lines.

Reply:
Ah, nevermind I see that there are multiple "migrating to x.y" in each version, so my suggestion wouldn't solve the problem here.